### PR TITLE
Update wiki cli 0.5

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

With the update of the wiki from GitHub Pages to vercel we broke all relative links that don't use file extensions.
So I updated the cli (which from now on should detect those broken links again) and fixed already existing broken links.

Sorry for the inconvenience.

## Type of change

- Documentation Fix

## How the change has been tested

Tested locally and the docs test workflow should succeed too

## Change checklist

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by eike-hass — APPROVED** [review]

File: documentation/docs/specs/didcomm/protocols/authentication.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ This protocol allows two parties to mutually authenticate by disclosing and veri
 
 <div style={{textAlign: 'center'}}>
 
-<sub>For guidance on diagrams see the <a href="../overview.md#diagrams">corresponding section in the overview</a>.</sub>
```
Comment (eike-hass) [inline documentation/docs/specs/didcomm/protocols/authentication.md:38]: should this be with `.md`?

File: documentation/docs/specs/didcomm/protocols/connection.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Allows establishment of a [DIDComm connection](https://identity.foundation/didco
 
 <div style={{textAlign: 'center'}}>
 
-<sub>For guidance on diagrams see the <a href="../overview#diagrams">corresponding section in the overview</a>.</sub>
```
Comment (eike-hass) [inline documentation/docs/specs/didcomm/protocols/connection.md:41]: should this be with .md?

File: documentation/docs/specs/didcomm/protocols/issuance.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Allows a [holder](#roles) to request a [verifiable credential][VC] from an [issu
 
 <div style={{textAlign: 'center'}}>
 
-<sub>For guidance on diagrams see the <a href="../overview#diagrams">corresponding section in the overview</a>.</sub>
+<sub>For guidance on diagrams see the <a href="../../overview#diagrams">corresponding section in the overview</a>.</sub>
```
Comment (eike-hass) [inline documentation/docs/specs/didcomm/protocols/issuance.md:40]: should this be with `.md`?

File: documentation/docs/specs/didcomm/protocols/post.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ Allows the sending of a single message with arbitrary data. Multiple [post](#pos
 
 <div style={{textAlign: 'center'}}>
 
-<sub>For guidance on diagrams see the <a href="../overview#diagrams">corresponding section in the overview</a>.</sub>
+<sub>For guidance on diagrams see the <a href="../../overview#diagrams">corresponding section in the overview</a>.</sub>
```
Comment (eike-hass) [inline documentation/docs/specs/didcomm/protocols/post.md:38]: should this be with `.md`?

File: documentation/docs/specs/didcomm/protocols/presentation.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ Allows presentation of one or more [verifiable credentials](https://www.w3.org/T
 
 <div style={{textAlign: 'center'}}>
 
-<sub>For guidance on diagrams see the <a href="../overview#diagrams">corresponding section in the overview</a>.</sub>
+<sub>For guidance on diagrams see the <a href="../../overview#diagrams">corresponding section in the overview</a>.</sub>
```
Comment (eike-hass) [inline documentation/docs/specs/didcomm/protocols/presentation.md:40]: should this be with `.md`?

File: documentation/docs/specs/didcomm/protocols/revocation-options.md [hunk 6]
```diff
@@ -30,7 +30,7 @@ Allows discovery of available [`RevocationInfo`](./revocation.md#RevocationInfo)
 
 <div style={{textAlign: 'center'}}>
 
-<sub>For guidance on diagrams see the <a href="../overview#diagrams">corresponding section in the overview</a>.</sub>
+<sub>For guidance on diagrams see the <a href="../../overview#diagrams">corresponding section in the overview</a>.</sub>
```
Comment (eike-hass) [inline documentation/docs/specs/didcomm/protocols/revocation-options.md:33]: should this be with `.md`?

**Comment by eike-hass** [comment]
I stopped commenting on the files, but should the `overview` links all contain the `.md` extension as well?

File: documentation/docs/specs/didcomm/protocols/authentication.md (hunk 1 above)
Comment (Dr-Electron) [reply documentation/docs/specs/didcomm/protocols/authentication.md:38]: Html is not transformed by mdx. So it‘s not needed here.

**Comment by Dr-Electron** [comment]
> I stopped commenting on the files, but should the `overview` links all contain the `.md` extension as well?

As said above this is related on how Docusaurus handles markdown links. Htmls tags are mostly ignored

**Pull request merged**